easybuild/easyconfigs/b/BuildEnv/BuildEnv-1.0-intel-2015a-default.eb

```diff
@@ -0,0 +1,14 @@
+easyblock = "BuildEnv"
+
+name = 'BuildEnv'
+version = '1.0'
```

Comment:
will there ever be another version? (I'm not saying I have a better option for this, right now)